driver.go

```diff
@@ -66,6 +74,7 @@ func (a *Driver) Open(dsn string) (driver.Conn, error) {
 		keytab:              config.keytab,
 		krb5Conf:            config.krb5Conf,
 		krb5CredentialCache: config.krb5CredentialCache,
+		additionalHeaders:   c.additionalHeaders,
```

Comment:
We don't need this since the user can pass in his own `http.Client`.

Reply:
done